src/modules/iiif-viewer/IiifViewer.tsx

```diff
@@ -132,9 +134,11 @@ const IiifViewer = forwardRef<IiifViewerFunctions, IiifViewerProps>(
 				// console.log(altoTextLocations);
 
 				if (isBrowser()) {
-					altoTextLocations.forEach((altoTextLocation) => {
+					altoTextLocations.forEach((altoTextLocation, index) => {
 						const span = document.createElement('SPAN');
-						span.className = 'p-object-detail__iiif__alto__text';
+						span.className =
+							'p-object-detail__iiif__alto__text p-object-detail__iiif__alto__text__' +
+							index;

```

Comment:
use an `id` or [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes#javascript_access) to target elements

Reply:
☝️ non-optional IMO

Reply:
done